README.md

```diff
@@ -49,6 +49,45 @@ $ docker run --name ecs-agent \
     amazon/amazon-ecs-agent:latest
 ```
 
+### On Other Linux AMIs when awsvpc networking mode is enabled
+
+For the AWS VPC networking mode, ECS agent requires CNI plugin and dhclient to be available. ECS also needs the ecs-init to run as part of its startup.
+The following is an example of docker run configuration for running ecs-agent with Task ENI enabled
+```
+$ # Run the agent
+$ /usr/bin/docker run --name %i \
+--init \
+--privileged \
+--restart=on-failure:10 \
+--volume=/var/run:/var/run \
+--volume=/var/log/ecs/:/log:Z \
+--volume=/var/lib/ecs/data:/data:Z \
+--volume=/etc/ecs:/etc/ecs \
+--volume=/lib:/lib \
+--volume=/sbin:/sbin \
+--volume=/lib64:/lib64 \
+--volume=/usr/lib:/usr/lib \
+--volume=/proc:/host/proc \
+--volume=/sys/fs/cgroup:/sys/fs/cgroup \
+--volume=/var/lib/ecs/dhclient:/var/lib/dhclient \
+--net=host \
+--env-file=/etc/ecs/ecs.config \
+--cap-add=sys_admin \
+--cap-add=net_admin \
+--env ECS_ENABLE_TASK_ENI=true \
+--env ENI_DHCLIENT_LEASES_PATH=/usr/sbin/dhclient \
+--env ECS_UPDATES_ENABLED=true \
+--env ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION=1h \
+--env ECS_ENABLE_CONTAINER_METADATA=false \
+--env ECS_DATADIR=/data \
+--env ECS_ENABLE_TASK_IAM_ROLE=true \
+--env ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST=true \
+--env ECS_LOFILE=/log/ecs-agent.log \
```

Comment:
```suggestion
--env ECS_LOGFILE=/log/ecs-agent.log \
```

Reply:
@the0ffh , good find! Thank you. I have fixed this. Please review